source/_integrations/cover.rflink.markdown

```diff
@@ -168,6 +168,38 @@ cover:
 
 The configuration above shows that the `type` property may be omitted. When the ID starts with `newkaku`, the component will make sure that the on and off commands are inverted. When the ID does not start with `newkaku`, the on and off commands are not inverted.
 
+## Setting up a Dooya, Zemismart or Brel motor device
```

Comment:
I think it is better not to refer to a specific cover model in the title, since this information applies to any NON-RTS shutter.

I suggest you replace the tittle and add the information just before the `## Setting up a KAKU ASUN-650 device` section?

Reply:
```suggestion
## Setting up a non-RTS cover
```
Agreed